sentry-core/src/test/java/io/sentry/core/SentryOptionsTest.kt

```diff
@@ -124,4 +124,28 @@ class SentryOptionsTest {
         val options = SentryOptions()
         assertNotNull(options.executorService)
     }
+
+    @Test
+    fun `init should set SdkVersion`() {
+        val sentryOptions = SentryOptions()
+        assertNotNull(sentryOptions.sdkVersion)
+        val sdkVersion = sentryOptions.sdkVersion!!
+
+        assertEquals(BuildConfig.SENTRY_CLIENT_NAME, sdkVersion.name)
+        assertEquals(BuildConfig.VERSION_NAME, sdkVersion.version)
+
+        assertTrue(sdkVersion.packages!!.any {
```

Comment:
Can we have a test like this for the android package?
Would be nice to have one fro the `ndk` package too

Reply:
We have these tests already:
- NDK - https://github.com/getsentry/sentry-android/blob/master/sentry-android-ndk/src/test/java/io/sentry/android/ndk/SentryNdkUtilTest.kt#L12
- Android - https://github.com/getsentry/sentry-android/blob/master/sentry-android-core/src/test/java/io/sentry/android/core/SentryAndroidOptionsTest.kt#L20